.travis/test_04_install.sh

```diff
@@ -7,7 +7,8 @@
 export LC_ALL=C.UTF-8
 
 travis_retry docker pull "$DOCKER_NAME_TAG"
-env | grep -E '^(CCACHE_|WINEDEBUG|LC_ALL|BOOST_TEST_RANDOM|CONFIG_SHELL)' | tee /tmp/env
+export ASAN_OPTIONS="check_initialization_order=1"
```

Comment:
I think this enviroment variable can be moved to [.travis.yml](../../tree/master/.travis.yml)